src/Nox.Types.EntityFramework/Types/MacAddress/MacAddressDatabaseConfigurator.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+using Nox.Solution;
+using Nox.Types.EntityFramework.Abstractions;
+
+namespace Nox.Types.EntityFramework.Types;
+
+public class MacAddressDatabaseConfigurator : INoxTypeDatabaseConfigurator
+{
+    public NoxType ForNoxType => NoxType.MacAddress;
+
+    public bool IsDefault => true;
+
+    public void ConfigureEntityProperty(
+        NoxSolutionCodeGeneratorState noxSolutionCodeGeneratorState, 
+        EntityTypeBuilder builder, 
+        NoxSimpleTypeDefinition property, 
+        Entity entity, 
+        bool isKey)
+    {
+        builder
+            .Property(property.Name)
+            .IsRequired(isKey || property.IsRequired)
+            .IsUnicode(false)
+            .IsFixedLength(true)
+            .HasMaxLength(12)
+            .HasConversion<MacAddressConverter>();
```

Comment:
MacAddressConverter should use "FromDatabase"

Reply:
Fixed, thanks